### PR TITLE
apps/system/utils/heapinfo : Support -i option for initializing the p…

### PR DESCRIPTION
**Pull request by jeongchanKim**
…eak size

"heapinfo -i" initializes the peak allocated memory size in each heap.
It can be used for measuring the peak size from that time on.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [2c86b627293787859fc1606b617a0be86d4365e0] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2c86b627293787859fc1606b617a0be86d4365e0] - Code Rule Check (C++) OK.

File: apps/system/utils/utils_heapinfo.c [hunk 1]
```diff
@@ -214,11 +215,17 @@ int utils_heapinfo(int argc, char **args)
...
 	}
 
-	while ((opt = getopt(argc, args, "kub:ap:fge:r")) != ERROR) {
+	while ((opt = getopt(argc, args, "ikub:ap:fge:r")) != ERROR) {
 #if CONFIG_MM_NHEAPS > 1
 		summary_option = false;
 #endif
 		switch (opt) {
+		/* i : initialize the peak allocated memory size. */
+			break;
```
Comment (sunghan-chang) [inline apps/system/utils/utils_heapinfo.c]: please check this.

**Comment by seinfra** [comment]
Target : [8618adf6b4852a1f6d90fad52b4b1445eea0cc02] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8618adf6b4852a1f6d90fad52b4b1445eea0cc02] - Code Rule Check OK.

**Pull request merged**